config/deploy/production.rb

```diff
@@ -7,23 +7,21 @@
 require 'capistrano/gitflow_version'
 
 # define servers and their roles and attributes
-server "ao3-app01.ao3.org",  :app , :db
-server "ao3-app98.ao3.org",  :app , :workers , :schedulers
-server "ao3-app02.ao3.org",  :app , primary: true
+server "ao3-app01.ao3.org",  :app , :db, :workers , :schedulers
```

Comment:
Is the space before the comma necessary? If so, it's missing on `:db,` here and `:schedulers,` below. If not, it might be a good idea to get rid of it, as Hound suggests.